cmd/api/api.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/gitpods/gitpods/authorization"
 	"github.com/gitpods/gitpods/cmd"
+	apiv1 "github.com/gitpods/gitpods/internal/api/v1"
```

Comment:
I would just import this as `api "..."` (we could change it if we get to v2)

Reply:
IMHO it's a good convention to directly clearly define it as v1

Reply:
As this is just the alias I don't have a strong opinion on this, but I think that the apiv1 alias is already the future proof solution. I'll simply leave it like this.